dd-sdk-android-core/api/apiSurface

```diff
@@ -177,6 +177,7 @@ data class com.datadog.android.core.configuration.Configuration
     fun setAdditionalConfiguration(Map<String, Any>): Builder
     fun setProxy(java.net.Proxy, okhttp3.Authenticator?): Builder
     fun setEncryption(com.datadog.android.security.Encryption): Builder
+    fun setPersistenceStrategyFactory(com.datadog.android.core.persistence.PersistenceStrategy.Factory?): Builder
```

Comment:
Should we hide it behind some internal proxy? I was thinking that we are not going to show this method to everyone.

Reply:
The thing is that we don't have an internal proxy for the configuration. The fact that the interface to implement is not trivial at all makes me think that no one (if they discover it) will bother implementing it